(ForumsTeam) Renamed the link name

kadamajay5 opened the pull request:
Under Windows client section "by using automation" link is taking to TechNet forum thread. I feel either we should change the link with appropriate article/blog/doc or we can change the name of link as by using automation as discussed here. I have changed the link name accordingly. Kindly review it once and do the needful.

azurecla commented:
Hi __@dazfuller__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@emgerner-msft - Please review this pull request from @dazfuller  in the public repository and let me know if the changes should be merged.

emgerner-msft commented:
@tysonn JC fixed this in the internal content repo 4 days ago and it was merged and published. The [python blob article](https://azure.microsoft.com/en-us/documentation/articles/storage-python-how-to-use-blob-storage/) looks correct. The change for some reason isn't in the public repo though. Merging this could cause some nasty merge issues depending on the process you use...

dazfuller commented:
@tysonn Just caught up on this, the public repo and the article look fine now, I'm guessing I must have submitted a change around the same time as the article was being updated.  I'll close this pull request by tomorrow morning unless anyone wants to hang on to it.

tysonn commented:
@dazfuller - Thanks, I've closed.

kadamajay5 committed:
(ForumsTeam) Renamed the link name

Under Windows client section "by using automation" link is taking to TechNet forum thread. I feel either we should change the link with appropriate article/blog/doc or we can change the name of link as by using automation as discussed here. I have changed the link name accordingly. Kindly review it once and do the needful.
